domain-management/src/main/java/org/jboss/as/domain/management/connections/ldap/LdapConnectionAddHandler.java

```diff
@@ -125,7 +127,17 @@ static Config updateRuntime(final OperationContext context, final ModelNode mode
             handlesReferralsForSet = Collections.emptySet();
         }
 
-        return connectionManagerService.setConfiguration(initialContextFactory, url, searchDn, searchCredential, referralHandling, handlesReferralsForSet);
+        final boolean alwaysSendClientCert;
+        if (model.hasDefined(ALWAYS_SEND_CLIENT_CERT.getName())) {
+            /* If always-send-client-cert is defined in the management model, take it */
+            alwaysSendClientCert = ALWAYS_SEND_CLIENT_CERT.resolveModelAttribute(context, model).asBoolean();
```

Comment:
given you have default value defined on attribute all this should be simplified to
`boolean alwaysSendClientCert = ALWAYS_SEND_CLIENT_CERT.resolveModelAttribute(context, model).asBoolean();`

and that is it.